data/dev.js

```diff
@@ -1,31 +1,34 @@
+/* please have the dev team director as the first element of this array, so that the 
+dev team page correctly autopopulates the director's data! */
 export const people = [
-  {
-		name: 'Advaith Gowrishetty',
+//dev team director also has an extra email field as they are the point of contact for the dev team
```

Comment:
Do you want to encode this somewhere else instead of in this field? Ex a constant somewhere? I feel like this complicates the `people` design.

Reply:
Also agree with this ^ would be easier to just have a constant in the page

Reply:
Sure, this can be set as a constant within the `dev.js` page instead